src/array.rs

```diff
@@ -99,8 +99,9 @@ macro_rules! small_array {
             }
         }
 
-        impl<S : Strategy> Strategy for [S;$n] {
-            type Value = ArrayValueTree<[S::Value;$n]>;
+        impl<S : Strategy> Strategy for [S; $n] {
+            type Tree = ArrayValueTree<[S::Tree; $n]>;
+            type Value = [ValueFor<S>; $n];
```

Comment:
Isn't `ValueFor` essentially obsolete now?

Reply:
Kinda; but keeping it around causes less breakage? And if we make future changes, then the alias helps reduce it further.

But I'll remove it if you want.